components/backup/src/writer.rs

```diff
@@ -79,34 +83,36 @@ impl BackupWriter {
     /// Save buffered SST files to the given external storage.
     pub fn save(mut self, storage: &dyn ExternalStorage) -> Result<Vec<File>> {
         let name = self.name;
-        let save_and_build_file = |cf, mut contents: &[u8]| -> Result<File> {
-            BACKUP_RANGE_SIZE_HISTOGRAM_VEC
-                .with_label_values(&[cf])
-                .observe(contents.len() as _);
-            let name = format!("{}_{}.sst", name, cf);
-            let checksum = tikv_util::file::calc_crc32_bytes(contents);
-            storage.write(&name, &mut contents as &mut dyn std::io::Read)?;
-            let mut file = File::new();
-            file.set_crc32(checksum);
-            file.set_name(name);
-            Ok(file)
-        };
+        let save_and_build_file =
+            |cf, mut contents: &[u8], limiter: Option<&Arc<IOLimiter>>| -> Result<File> {
```

Comment:
Since the `limiter` is always `.cloned()` why not just pass `Option<Arc<IOLimiter>>` here?